spec/lib/miq_preloader_spec.rb

```diff
@@ -4,22 +4,35 @@
       ems = FactoryBot.create(:ems_infra)
       expect(ems.vms).not_to be_loaded
       expect { preload(ems, :vms) }.to make_database_queries(:count => 1)
+
       expect(ems.vms).to be_loaded
       expect { preload(ems, :vms) }.to_not make_database_queries
+      expect { ems.vms.size }.not_to make_database_queries
```

Comment:
Keeping consistent with the line before it.  (although it's not_to on line 5 also, so 🤷 )

```suggestion
      expect { ems.vms.size }.to_not make_database_queries
```

Reply:
I `s/to_not/not_to/g` to make consistent.

Other places said not to split the participle (to verb)